### DISCUSSION TITLE
issue with importing video

### DISCUSSION THREAD

**Discussion by CowlBlue** [keiwando/toonsquid#583] [General]
Hello!  recently I noticed an issue whenever I import a video where the frames of the video were offset by one frame (the first frame being duplicated).  I've demonstrated the issue in the video below:

https://github.com/keiwando/toonsquid/assets/167805438/b3c0e08d-cf0d-4b82-b504-1f36e593a0be

**Comment by kavlor** [comment]
I just tried this and got the same . Seems some apps import correctly and some do the same as Toonsquid. Imported into Rough Animator and you get the correct frames but import into Lumafusion and you get the same issue where its 11234.

**Comment by keiwando** [comment]
Hi!
This is actually caused by a bug in iOS itself which sometimes changes the frame rates of video assets (particularly with such short videos) when they are imported into an app under certain circumstances. If you open the library, it probably shows the imported video as 20fps. The current workaround would be to edit the animation clip of the imported video and change its frame rate back to the correct value (in this case 24fps). Then, the frame numbers should be correct again in the main timeline. Theoretically, there are some other rare issues that such a manual frame rate change could cause for the video playback, but this is the best you could do right now if you run into this.

I have already reported this bug to Apple but haven't gotten any response about it so far. I will also be evaluating potential other options for importing the video that hopefully avoid this issue, but most alternatives have severe trade-offs as well, so it's unfortunately not an easy fix.

**Reply by CowlBlue** [reply]
Ah gotcha!
